boards/arm/rpi_pico/doc/index.rst

```diff
@@ -28,6 +29,7 @@ Hardware
 - 8 Programmable I/O (PIO) for custom peripherals
 - On-board LED
 - 1 Watchdog timer peripheral
+- Infineon CYW43439 2.4 GHz Wi-Fi chip (Pico W only)
 
 
 .. figure:: img/rpi_pico.jpg
```

Comment:
You don't need an image if you don't reference it from index.rst.

Reply:
Added a reference.